src/brushes/eraser_brush.class.js

```diff
@@ -0,0 +1,592 @@
+(function () {
+  var _proto = fabric.util.object.clone(fabric.StaticCanvas.prototype);
```

Comment:
why do we need to clone the prototype? can't we just reference it?

Reply:
Because it creates a circular ref after it's extended